lms/djangoapps/course_home_api/progress/tests/test_views.py

```diff
@@ -184,6 +192,60 @@ def test_learner_has_access(self):
         assert not gated_score['learner_has_access']
         assert ungated_score['learner_has_access']
 
+    @override_waffle_flag(COURSE_HOME_MICROFRONTEND_PROGRESS_TAB, active=True)
+    @override_waffle_flag(waffle_flags()[WRITABLE_GRADEBOOK], active=True)
+    @patch.dict(settings.FEATURES, {'ASSUME_ZERO_GRADE_IF_ABSENT_FOR_ALL_TESTS': False})
+    @patch.dict(settings.FEATURES, {'PERSISTENT_GRADES_ENABLED_FOR_ALL_TESTS': True})
+    def test_persistent_grade(self):
```

Comment:
nit: maybe worth adding a "no override serializes as None" test (or add an assert to an existing generic test) for the common "no override" case